pyne/endf.pyx

```diff
@@ -175,27 +181,24 @@ class Library(rxdata.RxLib):
         mf = 1
         start = (self.chars_til_now+self.offset)//self.line_length
         stop = start  # if no 451 can be found
+        line = fh.readline() # get the next line; start parsing from the 6th line
         while FILE1_R.search(line):
+            # divide 1-66 chars of the line into six 11-char parts
+            lineparts = [line[i:i+11] for i in range(0, 66, 11)]
             # parse contents section
-            if CONTENTS_R.match(line):
+            if self._isContentLine(lineparts):
                 # When MF and MT change, add offset due to SEND/FEND records.
                 old_mf = mf
-                mf, mt = int(line[22:33]), int(line[33:44])
+                mf, mt = int(lineparts[2]), int(lineparts[3])
                 if old_mf != mf:
                     start += 1
-                mt_length = int(line[44:55])
+                mt_length = int(lineparts[4])
                 stop = start + mt_length
                 self.mat_dict[nuc]['mfs'][mf, mt] = (self.line_length*start-self.offset,
                                                      self.line_length*stop-self.offset)
                 start = stop + 1
                 line = fh.readline()
             # parse comment
-            elif SPACE66_R.match(line):
-                self.structure[nuc]['docs'].append(line[0:66])
-                line = fh.readline()
-            elif NUMERICAL_DATA_R.match(line):
```

Comment:
Why don't we need this case any more?

Reply:
My understanding is that there're three parts in a (MF,MT)=(1,451) file, the data part which comes first, then the document part, and at last the contents/dictionary part.  As the meaningful values in the data part has already read with fh.read(3*len(line)) (and I don't think we have to go back to the top to read the same data lines again), we only have to distinguish the contents lines from others, i.e. 'docs' lines.

Reply:
Ah, according to the [manual](https://www.nndc.bnl.gov/csewg/docs/endf-manual.pdf#page=54), there're only two parts, 1. a documentation, and 2. a dictionary, and 

> The first three records of the descriptive information contain a standardized presentation
of information on the material, projectile, evaluators, and modification status.